pkg/activator/handler/handler.go

```diff
@@ -84,12 +91,23 @@ func (a *ActivationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		Host:   host,
 	}
 
+	// Setup tracing transport
+	tr := a.Transport
+	traceTr, err := zipkinhttp.NewTransport(tracer, zipkinhttp.RoundTripper(tr))
+	if err != nil {
+		a.Logger.Errorw("Failed to create zipkin http transport", zap.Error(err))
+	} else {
+		tr = traceTr
+	}
+
 	err = a.Throttler.Try(revID, func() {
 		var (
 			httpStatus int
 			attempts   int
 		)
 
+		probeSpan, reqCtx := tracer.StartSpanFromContext(r.Context(), "probe")
```

Comment:
what if move this into its own function then
1. this function will become more readable
2. we can do `probespan, ... defer probespan.finish()`